src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -1,147 +0,0 @@
-```
-
-from the project's root directory. New package sources must be added after the `<clear />` element if you decide to keep it.
-
-Once you have added the package sources, add a reference to the ILCompiler package either by running
-```bash
-> dotnet add package Microsoft.DotNet.ILCompiler -v 10.0.0-*
-```
-
-or by adding the following element to the project file:
-```xml
-<ItemGroup>
-  <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="10.0.0-*" />
-</ItemGroup>
-```
-
-Adding an explicit package reference to `Microsoft.DotNet.ILCompiler` will generate warning when publishing and it can run into version errors. When possible, use the default `Microsoft.DotNet.ILCompiler` version to publish a native AOT application.
-
-## Cross-architecture compilation
-
-Native AOT toolchain allows targeting ARM64 on an x64 host and vice versa for both Windows and Linux. Cross-OS compilation, such as targeting Linux on a Windows host, is not supported.
-
-The target architecture can be specified using `-r` or `--arch` options of the `dotnet publish` command. For example, the following command produces Windows Arm64 binary on a Windows x64 host machine:
-
-```bash
-> dotnet publish -r win-arm64
-```
-
-The cross-architecture compilation requires native build tools for the target platform to be installed and configured correctly. On Linux, you may need to follow [cross-building instructions](../../../../docs/workflow/building/coreclr/cross-building.md) to create your own sysroot directory and specify path to it using the `SysRoot` property.
-
-See [Building native AOT apps in containers](containers.md) for a streamlined path to establishing a Linux cross-compilation environments.
-
-#### Using daily builds with cross-architecture compilation
-
-For using daily builds according to the instructions above, in addition to the `Microsoft.DotNet.ILCompiler` package reference, also add the `runtime.win-x64.Microsoft.DotNet.ILCompiler` package reference to get the x64-hosted compiler:
-```xml
-<PackageReference Include="Microsoft.DotNet.ILCompiler; runtime.win-x64.Microsoft.DotNet.ILCompiler" Version="9.0.0-alpha.1.23456.7" />
-```
-
-Replace `9.0.0-alpha.1.23456.7` with the latest version from the [dotnet10](https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet10/NuGet/Microsoft.DotNet.ILCompiler/) feed.
-Note that it is important to use _the same version_ for both packages to avoid potential hard-to-debug issues. After adding the package reference, you may publish for win-arm64 as usual:
-```bash
-> dotnet publish -r win-arm64
-```
-
-Similarly, to target linux-arm64 on a Linux x64 host, in addition to the `Microsoft.DotNet.ILCompiler` package reference, also add the `runtime.linux-x64.Microsoft.DotNet.ILCompiler` package reference to get the x64-hosted compiler:
-```xml
-<PackageReference Include="Microsoft.DotNet.ILCompiler; runtime.linux-x64.Microsoft.DotNet.ILCompiler" Version="9.0.0-alpha.1.23456.7" />
-```
-
-## Using statically linked ICU
```

Comment:
We do not have the information about statically linked ICU and OpenSSL in the official docs since it is officially unsupported. This should stay here since these docs are describing officially unsupported features.